Resources/ServerInfo/Guidebook/_DV/NewPlayer/Whitelist.xml

```diff
@@ -0,0 +1,6 @@
+<Document>
+# Whitelist
+A portion of roles on DeltaV servers - primarily [textlink="Security" link="Security"] and [textlink="Command" link="Command"] roles - are locked behind an in-game whitelist; you will not be able to play these roles unless you successfully apply for them.
+
+To apply, follow the steps in the [textlink="whitelisting forum on our Discord" link="https://discord.com/channels/968983104247185448/1387670020376625233"], or send an email to [textlink="trianglevee@aol.com" link="mailto:trianglevee@aol.com?subject=Whitelist%20Application"] containing your username and the whitelist tier you are applying for.
```

Comment:
one might think all you need is to email your username and e.g. command with nothing else